corehq/apps/locations/models.py

```diff
@@ -800,3 +800,16 @@ def _unassign_users_from_location(domain, location_id):
             user.unset_location_by_id(domain, location_id, fall_back_to_next=True)
         elif user.is_commcare_user():
             user.unset_location_by_id(location_id, fall_back_to_next=True)
+
+
+class LocationGroup(models.Model):
```

Comment:
Could you add some docs about what this is and how it's used?

Reply:
https://github.com/dimagi/commcare-hq/pull/21349/commits/bb23878d009fb6cbacf7975bcea3f419af866afc